laravel/the-basics/databases.html.md.erb

```diff
@@ -202,43 +194,45 @@ If you decide not to use the `flyctl postgres attach` command above, and would w
 Manually connecting with your Postgres Fly App also means manually creating your Postgres Database. 
 You can interact with your Postgres Fly App using `flyctl postgres connect`. 
 
-1) Connect with your Postgres Fly App using its App name. 
-
-```cmd
-flyctl postgres connect -a <postgres-fly-app>
-```
-```output
-Connecting to <postgres-fly-app>.internal... complete
-psql (14.4 (Debian 14.4-1.pgdg110+1))
-Type "help" for help.
-
-postgres=# 
-```
-<small>In case you've forgotten about your app name, but managed to save the output from the Postgres Fly App creation (which you should've!), then check the value of the `Hostname`. The App name would simply be every little character before ".internal".</small> 
-
-
-2) Great, you're connected! Go ahead and complete your setup with a new database:
-```postgres
-create database <DB_NAME>;
-```
-
-3) Now if you check your database list, you should find the recent database created:
-```sql
-SELECT datname FROM pg_database;
-```
-```output
-  datname  
------------
- postgres
- testdb
- template1
- template0
-(4 rows)
-```
+1.  Connect with your Postgres Fly App using its App name. 
+    ```cmd
+    flyctl postgres connect -a <postgres-fly-app>
+    ```
+    ```output
+    Connecting to <postgres-fly-app>.internal... complete
+    psql (14.4 (Debian 14.4-1.pgdg110+1))
+    Type "help" for help.
+
+    postgres=# 
+    ```
+
+    <small>In case you've forgotten about your app name, but managed to save the output from the Postgres Fly App creation (which you should've!), then check the value of the `Hostname`. The App name would simply be every little character before ".internal".</small>
+    <br /> 
+    <br /> 
+2.  Great, you're connected! Go ahead and complete your setup with a new database:
+    ```postgres
+    create database <DB_NAME>;
+    ```
+
+3.  Now if you check your database list, you should find the recent database created:
+    
+    ```cmd
+    SELECT datname FROM pg_database;
+    ```
+    ```output
+      datname  
+    ~-----------
```

Comment:
this is a load bearing `~` we cannot change this because otherwise a customer renderer thinks its a changelog items and it totally ruins the formatting of the entire page in spectacular fashion.